src/pages/dashboard/_StateCumulativeTestsContainer.js

```diff
@@ -98,19 +128,26 @@ export default function CumulativeTestsByStateContainer() {
             state
             positive
             negative
+            totalTestResults
           }
         }
       }
     }
   `)
 
-  const data = groupAndSortStateDaily(query)
+  const [useTestsPerCapita, setUseTestsPerCapita] = useState(false)
+
+  const toggleChartData = () => setUseTestsPerCapita(u => !u)
+
+  const allData = useMemo(() => groupAndSortStateDaily(query), [query])
+
+  const data = useMemo(() => {
+    return useTestsPerCapita ? allData.perCapita : allData.totals
+  }, [useTestsPerCapita])
 
   const maxStateTests = useMemo(() => {
-    return max(data, stateData => {
-      return max(stateData.values, d => calculateTotal(d))
-    })
-  })
+    return data[0].values[0].totalTestResults
+  }, [useTestsPerCapita])
```

Comment:
i would use `data` here instead of `useTestsPerCapita` since that's the variable that's used to produce this value.  That said the end result is no different so i'm not going to worry about this one.